docker/test/stateless/setup_minio.sh

```diff
@@ -1,29 +1,40 @@
 #!/bin/bash
 
-# Usage for local run:
-#
-# ./docker/test/stateless/setup_minio.sh ./tests/
-#
+USAGE='Usage for local run:
+
+./docker/test/stateless/setup_minio.sh { stateful | stateless } ./tests/
+
+'
 
 set -e -x -a -u
 
-rpm2cpio ./minio-20220103182258.0.0.*.rpm | cpio -i --make-directories
-find / -name minio
-cp ./usr/local/bin/minio ./
+TEST_TYPE="$1"
+shift
+
+case $TEST_TYPE in
+  stateless) QUERY_DIR=0_stateless ;;
+  stateful) QUERY_DIR=1_stateful ;;
+  *) echo "unknown test type $TEST_TYPE"; echo "${USAGE}"; exit 1 ;;
+esac
 
 ls -lha
 
 mkdir -p ./minio_data
 
 if [ ! -f ./minio ]; then
+  MINIO_VERSION=${MINIO_VERSION:-2022-01-03T18-22-58Z}
```

Comment:
Why it is better to have ENV in Dockerfile if there is a default value here?

Reply:
it's for the local running, when the binary is not found

Reply:
I mean if we will remove ENV from Dockerfile the result will be the same?

Reply:
They are unrelated envs. It even can be an ARG in Dockerfile, since mainly it's used for downloading `minio` during the build.

Reply:
I replaced them with ARG and added a decent comment